improver_tests/calibration/reliability_calibration/test_ApplyReliabilityCalibration.py

```diff
@@ -469,6 +469,53 @@ def test_calibrating_without_single_value_bins(self, warning_list=None):
         assert_allclose(result[1].data, expected_1)
         self.assertFalse(warning_list)
 
+    def test_calibrating_point_by_point(self, warning_list=None):
```

Comment:
Just to finish off this testing, I think that it might be good include, perhaps two additional unit tests using site forecasts, with one unit test using the point by point functionality and the other not. I think that unit tests with site forecasts might help to make the need clearer for the handling of the auxiliary coordinates in the `_point_by_point_calibration` method.

To create site forecast test cubes, you need to use `build_spotdata_cube` something like [in this file](https://github.com/metoppv/improver/blob/master/improver_tests/calibration/ensemble_calibration/helper_functions.py#L211). You could move the creation of the site forecast test cubes into a `setUp` method to avoid any duplicating between the unit tests.

Reply:
I've added two new tests which use spot forecast cubes, one which utilises point_by_point functionality and one which does not